### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by jacedang** [sgrvinod/a-PyTorch-Tutorial-to-Super-Resolution#8]
Hi author,
Thanks for providing this tutorial. However, I didn't see your loss function in the models.py
Can you shed light on it? 
Thanks very much
